GooglePlayInstant/Editor/AppBundleBuilder.cs

```diff
@@ -25,56 +25,51 @@ public static class AppBundleBuilder
     {
         private const string BaseModuleZipFileName = "base.zip";
 
+        /// <summary>
+        /// Build an app bundle at the specified path, overwriting an existing file if one exists.
+        /// </summary>
         public static void Build(string aabFilePath)
         {
-            var tempFilePath = Path.GetTempFileName();
-            Debug.LogFormat("Building Package: {0}", tempFilePath);
-            var buildPlayerOptions = PlayInstantBuilder.CreateBuildPlayerOptions(tempFilePath, BuildOptions.None);
+            var binaryFormatFilePath = Path.GetTempFileName();
+            Debug.LogFormat("Building Package: {0}", binaryFormatFilePath);
 
             // Do not use BuildAndSign since this signature won't be used.
-            if (!PlayInstantBuilder.Build(buildPlayerOptions))
+            if (!PlayInstantBuilder.Build(
+                PlayInstantBuilder.CreateBuildPlayerOptions(binaryFormatFilePath, BuildOptions.None)))
             {
                 // Do not log here. The method we called was responsible for logging.
                 return;
             }
 
-            CreateAppBundle(tempFilePath, aabFilePath);
-        }
-
-        // TODO: currently all processing is synchronous; consider moving to a separate thread
-        private static void CreateAppBundle(string inputFile, string aabFilePath)
-        {
+            // TODO: currently all processing is synchronous; consider moving to a separate thread
             try
             {
                 DisplayProgress("Running aapt2", 0.2f);
                 var workingDirectory = new DirectoryInfo(Path.Combine(Path.GetTempPath(), "play-instant-unity"));
-                if (workingDirectory.Exists)
-                {
-                    workingDirectory.Delete(true);
-                }
+                workingDirectory.Delete(true);
```

Comment:
I think DirectoryInfo.Delete works differently than Create, in that it does throw an error if the directory doesn't exist.